app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find_by(id: params[:id])
+    if @quote.nil?
+      render :error
```

Comment:
Sorry, I was mistaken - `find` will raise an error if the record isn't found. just using `find` is sufficient